crates/apub/src/api/test.rs

```diff
@@ -0,0 +1,42 @@
+use activitypub_federation::config::Data;
+use lemmy_api_common::context::LemmyContext;
+use lemmy_db_schema::{
+  source::{
+    instance::Instance,
+    local_user::{LocalUser, LocalUserInsertForm},
+    person::{Person, PersonInsertForm},
+  },
+  traits::Crud,
+};
+use lemmy_db_views::structs::LocalUserView;
+use lemmy_utils::error::LemmyResult;
+
+#[derive(Default)]
+pub struct TestUser {
+  pub name: String,
+  pub bio: Option<String>,
+  pub admin: bool,
+}
+
+impl TestUser {
+  pub async fn create(context: &Data<LemmyContext>, form: TestUser) -> LemmyResult<LocalUserView> {
```

Comment:
```suggestion
  pub async fn create(self, context: &Data<LemmyContext>) -> LemmyResult<LocalUserView> {
```

Then you can call it as `TestUser::default().create()`